literature_search.py

```diff
@@ -297,181 +324,227 @@ def get_abstract_relevance(lm, question, papers, batch_size=5):
     class args:
         # question = "The effect of cosmic birefringence on the CMB polarization measurement."
         # output = "./birefringence.json"
-        question = "recent progress on the use of faraday rotation modulator (FRM), which is a wide bandwidth linear polarization modulator made of solid-state polarization switches that are capable of modulation up to 10 kHz."
+        question = "How to evluate poultry heat stress level?"
         output = "./frm.json"
 
     odir = os.path.dirname(args.output)
     if not os.path.exists(odir):
         os.makedirs(odir)
-        
-    # model class: S > A > B > C > ...
+
+    # # model class: S > A > B > C > ...
+    # lm = {
+    #     'S': LM("openai/gpt-4o"),
+    #     'A': LM("openai/gpt-4o-mini"),
+    #     'B': LM("nousresearch/hermes-3-llama-3.1-405b:free"), # 8k context, 4k output
+    #     'C': LM("liquid/lfm-40b"),                            # 32k context, 32k output
+    # }
+
     lm = {
-        'S': LM("openai/gpt-4o"),
-        'A': LM("openai/gpt-4o-mini"),
-        'B': LM("nousresearch/hermes-3-llama-3.1-405b:free"), # 8k context, 4k output
-        'C': LM("liquid/lfm-40b"),                            # 32k context, 32k output
+        "A": OpenAI3P(
+            "codegeex-4", os.getenv("OPENAI3P_URL"), os.getenv("OPENAI3P_APIKEY")
+        ),
+        "S": OpenAI3P(
+            "glm-4-plus", os.getenv("OPENAI3P_URL"), os.getenv("OPENAI3P_APIKEY")
+        ),
     }
```

Comment:
happy to merge this if this part can be left unchanged. I think we can make these configurations into a separate config file in the future.